solution/solveintents.go

```
@@ -121,6 +172,8 @@ func (ei *IntentsHandler) SolveIntents() modules.BatchHandlerFunc {
 		ei.logger.Info("Received batch of UserOperations for solution", "number", len(modelUserOps))
 		for idx, op := range modelUserOps {
 			ei.logger.Info("Received UserOperation", "index", idx, "isIntent", op.HasIntent(), "operation", op.String())
+			// store the status
+			ei.UpdateProcessingStatusInDB(ctx.Batch[idx], "", pb.ProcessingStatus_PROCESSING_STATUS_RECEIVED)
```

golangci-lint (1.62.2)

Error return value of `ei.UpdateProcessingStatusInDB` is not checked

(errcheck)